dem/installation/installation-checklist.md

```diff
@@ -0,0 +1,51 @@
+---
+id: installation-checklist
+title: Installation checklist
+---
+
+# DEM installation checklist
+
+This list covers the requirements to ensure proper operation of DEM's different modules for a given site.
+
+# User journeys
+
+There is **nothing to install** because DEM connects to your application just like any regular user.
+
+However, depending on the security level on your site, you may need to allow our IPs so that your anti-bot system does not block our probes. If that's the case, you'll find the procedure here:
+
+[DEM IP addresses](./dem-ip-addresses.md)
+
+# Real User Monitoring
+
+Real User Monitoring installs like any marketing tag: by inserting a JavaScript tag. For the full procedure and installation instructions, see:
```

Comment:
The sentence sounds unfinished
"by inserting a JavaScript tag in each page's metadata" (if that's what is actually done)?